python/ray/autoscaler/local/defaults.yaml

```diff
@@ -1,103 +1,50 @@
-# An unique identifier for the head node and workers of this cluster.
-cluster_name: default
+# This configuration file is used internally
+# to fill default settings for on-prem Ray clusters
+# bootstrapped by the Ray autoscaler.
+# For annotated examples, see the example yamls in this directory.
 
-## NOTE: Typically for local clusters, max_workers == len(worker_ips).
+cluster_name: default
 
-max_workers: 0
+auth: {}
 
-# The autoscaler will scale up the cluster faster with higher upscaling speed.
-# E.g., if the task requires adding more nodes then autoscaler will gradually
-# scale up the cluster in chunks of upscaling_speed*currently_running_nodes.
-# This number should be > 0.
 upscaling_speed: 1.0
-
 idle_timeout_minutes: 5
 
-# This executes all commands on all nodes in the docker container,
-# and opens all the necessary ports to support the Ray cluster.
-# Empty string means disabled. Assumes Docker is installed.
+# Docker use not recommended: ray down will not clean up Docker state.
```

Comment:
Wait, `ray down` actually calls `docker stop` =)

Reply:
Right :) And I see docker run always gets an rm flag.
Let me get rid of the scary message..


For the docker stop to work properly in my experimental setup, I'd have to supply external ips for the workers... beh... (in theory not so bad to include another list) Maybe the external head ip setting was superfluous and I could have set up the experiment to run ray up from within the VPC. Will just leave that as is for now.